### PR TITLE
feedbacks rendering moved to the itemRunner

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: views/js/runner/api/itemRunner.js [hunk 1]
```diff
@@ -418,7 +418,13 @@ define([
                     responses = provider.getResponses.call(this);
                 }
                 return responses;
-           }
+           },
+
+            renderFeedbacks : function renderFeedbacks(feedbacks, itemSession, done){
```
Comment (krampstudio) [inline views/js/runner/api/itemRunner.js]: Please add a comment. It's not clear what to expect and how.
Also what's the difficulty to test this new method ?

**Comment by zagovorichev** [comment]
@krampstudio Update

File: views/js/test/runner/api/test.js [hunk 2]
```diff
@@ -595,5 +595,65 @@ define(['jquery', 'lodash', 'taoItems/runner/api/itemRunner', 'taoItems/test/run
             QUnit.start();
         }, 10);
     });
-});
 
+// itemRunner().renderFeedbacks()
```
Comment (krampstudio) [inline views/js/test/runner/api/test.js]: seems a forgotten comment

File: views/js/test/runner/api/test.js [hunk 3]
```diff
@@ -595,5 +595,65 @@ define(['jquery', 'lodash', 'taoItems/runner/api/itemRunner', 'taoItems/test/run
...
+    module('ItemRunner renderFeedbacks', {
+        teardown : function(){
+            //reset the provides
```
Comment (krampstudio) [inline views/js/test/runner/api/test.js]: typo

File: views/js/runner/api/itemRunner.js [hunk 4]
```diff
@@ -418,7 +418,20 @@ define([
                     responses = provider.getResponses.call(this);
                 }
                 return responses;
-           }
+           },
+
+            /**
+             * Render modalFeedbacks according to itemSession into the current item object
+             *
+             * @param feedbacks
```
Comment (krampstudio) [inline views/js/runner/api/itemRunner.js]: The most important is missing : what `feedback` is supposed to contain ? And `itemSession` ? Does the `done` callback have parameters ?

File: views/js/test/runner/api/test.js (hunk 2 above)
Comment (zagovorichev) [inline views/js/test/runner/api/test.js]: no it's like comments before `// itemRunner().on().off().trigger()`

Comment (krampstudio) [inline views/js/test/runner/api/test.js]: Anyway, we don't let code comment, use module and test description instead

**Pull request merged**